OracleDatabase/RAC/OracleRealApplicationClusters/samples/applypatch/19.3.0/patches/applyGridPatches.sh_opatchauto

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2017 Oracle and/or its affiliates. All rights reserved.
+# 
+# Since: January, 2017
+# Author: gerald.venzl@oracle.com
```

Comment:
I would add your name to the Authors list if you've made any modifications to Gerald's source.

Reply:
Ok. I will fix this.